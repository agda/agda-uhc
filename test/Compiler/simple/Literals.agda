{-# OPTIONS --universe-polymorphism #-}
module Literals where

open import Common.Nat
open import Common.Float
open import Common.Char
open import Common.String
open import Common.Unit
open import Common.IO

afloat : Float
afloat = 1.23

astring : String
astring = "abc"

achar : Char
achar = 'd'

anat : Nat
anat = 123

main : IO Unit
main =
  printFloat  afloat  ,,
  putStr      astring ,,
  printChar   achar   ,,
  printNat    anat    ,,
  putStrLn ""
