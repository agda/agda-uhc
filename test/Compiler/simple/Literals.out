EXECUTED_PROGRAM

ret > ExitSuccess
out > 1.230000abcd123
out >
